src/pages/dataTableUtilities/getPageInfoColumns.js

```diff
@@ -71,25 +71,25 @@ const PAGE_INFO_ROWS = (pageObject, dispatch, route) => ({
   theirRef: {
     title: `${pageInfoStrings.their_ref}:`,
     info: pageObject.theirRef,
-    onPress: () => dispatch(PageActions.openModal(MODAL_KEYS.THEIR_REF_EDIT, route)),
+    onPress: () => dispatch(ACTIONS.openModal(MODAL_KEYS.THEIR_REF_EDIT, route)),
```

Comment:
Could be wrong, but this is the action constant, not the action creator. If you import `BasePageActions` from `getPageActions`? (This is a mess, I'm sorry :( )

Reply:
Whoops, 789ac842411e8eb93fb69f38bf6d08f9efc8a0f7.